pages/common/gst-launch-1.0.md

```diff
@@ -0,0 +1,20 @@
+# gst-launch-1.0
+
+> Create gstreamer pipelines on the command line.
```

Comment:
```suggestion
> Create GStreamer pipelines.
```

Reply:
I think the the "on the command line" is important as most of Gstreamer documentation is C code. As in "departing from the usual tools, here's a command line tool. What's opinion?

Reply:
We are currently moving away from specifying "CLI for", "on the command line", etc in page descriptions as it is generally understood that `tldr` is for CLI/console commands. For GUI  pages, we make distinctions in command descriptions instead.

But in this context, I think the presence of "on the command line." is fine as it provides a distinction from other components. @acuteenvy wdyt?

Reply:
> I think the the "on the command line" is important as most of Gstreamer documentation is C code. As in "departing from the usual tools, here's a command line tool". What's opinion?

> But in this context, I think the presence of "on the command line." is fine as it provides a distinction from other components. @acuteenvy wdyt?

We're mostly documenting command-line tools here, maybe some GUI ones here and there, but never C libraries. All the "other components" are not documented in `tldr`. Therefore, explicitly stating that the page is about a command-line tool is redundant.

---

The official man page also doesn't say "on the command-line":
> `gst-launch-1.0` - build and run a GStreamer pipeline

```suggestion
> Build and run a GStreamer pipeline.
```